### PR TITLE
Types added for case flags

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
### Change description ###

Types added for case flags

### CODE REVIEW

File: ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/FlagLauncher.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.ccd.sdk.type;
+
+import org.springframework.stereotype.Component;
+
+@Component
```
Comment (linusnorton) [inline ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/FlagLauncher.java:5]: Should this be `@ComplexType`?

**Comment by linusnorton** [comment]
You should add some fields to the test definition to check the output is what you want and I think there are some additional fields in CaseLink that need to be added.

**Comment by ElifDenizEsea** [comment]
case link related changes are done by Santoshini, how can I add test definition? @linusnorton

**Comment by linusnorton** [comment]
Check uk.gov.hmcts.reform.fpl.CCDConfig - you'll need to add some fields to CaseData with the new types you've added

**Comment by ElifDenizEsea** [comment]
@linusnorton test definition added

**Review by linusnorton — APPROVED** [review]

**Review by spotstory — APPROVED** [review]

**Pull request merged**